### ISSUE TITLE
Does this still work ?

### ISSUE THREAD

**Issue by seanbperiod** [chris-pie/tumblr-safemode-bypass#1]
I have it loading on a tumblr site, but I get the safe mode screen.

**Comment by chris-pie** [comment]
Hey, I put this thing on hold because Tumblr was still changing stuff and I wanted it to settle down. For now all it does is that it lets you view posts marked sensitive if they show up on your dashboard/on the sidebar view by clicking at them. Those posts are missing controls for now, and if you're using XKit it will not see them. It's pretty useless, since you can just go to author's blog most of the time. I don't think there's an easy way to completely bypass safe mode through a script. The best workaround is just making another blog and lying about your date of birth. Unfortunate, but I can't do much about it.
Tl;dr: it doesn't really work,
